modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsPropsTransformer.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {createRenderURL, openSelectionModal} from 'frontend-js-web';
+
+import {ACTIONS} from './actions';
+
+export default function propsTransformer({
+	additionalProps,
+	portletNamespace,
+	...props
+}) {
+	return {
+		...props,
+		onActionButtonClick(event, {item}) {
+			const action = item.data?.action;
+
+			if (action) {
+				event.preventDefault();
+				ACTIONS[action](portletNamespace);
+			}
+		},
+		onPlusButtonClick() {
+			const {
+				basePortletURL,
+				mvcPath,
+				p_p_state,
+				passwordPolicyId,
+				passwordPolicyName,
+				tabs1,
+				tabs2,
+			} = additionalProps;
+
+			const url = createRenderURL(basePortletURL, {
+				mvcPath,
+				p_p_auth: Liferay.authToken,
+				p_p_state,
+				passwordPolicyId,
+				tabs1,
+				tabs2,
+			});
+
+			openSelectionModal({
+				multiple: true,
+				onSelect(result) {
+					if (result && result.item) {
+						const form = document[`${portletNamespace}fm`];
+
+						if (result.memberType == 'users') {
+							const addUserIdsInput = form.querySelector(
+								`#${portletNamespace}addUserIds`
```

Comment:
We can use `getElementById`. The same below, for organizations.

Reply:
no we can't this is a form

```js
var form = document.createElement('form');
form.getElementById // undefined
// or
HTMLFormElement.prototype.getElementById // undefined
```

Reply:
Sorry for not being precise, I meant 
```
document.getElementById(`${portletNamespace}addUserIds`)
```
 It should be a bit faster than `querySelector`, although I'm really not sure how it compares to `Element.querySelector`, with its reduced scope. In my opinion, the main reason to use it by default is to ensure / promote uniqueness of `id`s on a page. We have some issues with that, see for example [this discussion](https://github.com/liferay-frontend/liferay-portal/pull/726#discussion_r564546825).

Reply:
> Sorry for not being precise, I meant
> 
> ```
> document.getElementById(`${portletNamespace}addUserIds`)
> ```
> 
> It should be a bit faster than `document.querySelector`, although I'm really not sure how it compares to `Element.querySelector`, with its reduced scope. In my opinion, the main reason to use it by default is to ensure / promote uniqueness of `id`s on a page. We have some issues with that, see for example [this discussion](https://github.com/liferay-frontend/liferay-portal/pull/726#discussion_r564546825).

Reply:
> It should be a bit faster than document.querySelector
Yes that's true, here's a small [test](https://jsben.ch/yz9yV)(although, I tend not to trust these benchmark tools because it's not really clear how they are testing this)

Ok then, let's do with `document.getElementById`.

Reply:
Changed [here](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsManagementToolbarPropsTransformer.js)